client/keystore/src/lib.rs

```diff
@@ -276,11 +277,87 @@ impl Store {
 		buf.push(key_type + key.as_str());
 		Some(buf)
 	}
+
+	/// Signs a message with the private key that matches
+	/// the public key passed.
+	fn _sign_with(
+		&self,
+		id: KeyTypeId,
+		key: &CryptoTypePublicPair,
+		msg: &[u8],
+	) -> std::result::Result<Vec<u8>, String> {
```

Comment:
we already import `std`, so maybe we could just extend that import to also get `Result` and have a shorter version here and below? Preference ...

```suggestion
	) -> Result<Vec<u8>, String> {
```

Reply:
Result is already defined in the same file as an alias:
https://github.com/paritytech/substrate/blob/ab53994d74b0ad6983432050d540f834a8dc2c7f/client/keystore/src/lib.rs#L53

So i thought i would just use the full import path so that it's clear as to which one we're using here.